### PR TITLE
Add instructions for specifying version via BP_NGINX_VERSION

### PR DESCRIPTION
**Pull request by thitch97**
Signed-off-by: Brayan Henao <bhenao@vmware.com>

## Checklist

* [x] I have viewed, signed, and submitted the Contributor License Agreement.
* [ ] I have linked issue(s) that this PR should close using keywords or the Github UI (See [docs](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue))
* [ ] I have added an integration test, if necessary.
* [ ] I have reviewed the [styleguide](https://github.com/paketo-buildpacks/community/blob/main/STYLEGUIDE.md) for guidance on my code quality.

### CODE REVIEW

**Review by kvedurmu — APPROVED** [review]
Looks good. Could we cut a release of NGINX buildpack containing these changes before merging @paketo-buildpacks/web-servers-maintainers?

**Comment by thitch97** [comment]
> Looks good. Could we cut a release of NGINX buildpack containing these changes before merging @paketo-buildpacks/web-servers-maintainers?

https://github.com/paketo-buildpacks/nginx/releases/tag/v0.2.0 includes this change.

**Pull request merged**
